docs/resources/v2-conformance-statement.md

```diff
@@ -61,7 +61,7 @@ The following `Content-Type` header(s) are supported:
 - `multipart/related; type="application/dicom"`
 - `application/dicom`
 
-> Note: The Server will <u>not</u> coerce or replace attributes that conflict with existing data. All data will be stored as provided.
+> Note: The Server will <u>not</u> coerce or replace existing data on the DICOM binary. The DICOM binary will be stored as provided.
```

Comment:
Looking through the standard, `DICOM file` is probably the best term to use - and adding the `PS 3.10` part makes it extra clear. https://dicom.nema.org/medical/dicom/current/output/html/part18.html#sect_10.5.1.4

```suggestion
> Note: the Server will <u>not</u> coerce or replace data in the DICOM PS 3.10 file. The DICOM file will be stored as provided, except where otherwise noted.
```